packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -2,35 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import {
-  Link,
-  RouteComponentProps,
-  useLocation,
-  useNavigate,
-} from '@reach/router';
-import React, { useCallback, useState } from 'react';
+import { Link, useLocation, useNavigate } from '@reach/router';
+import React, { useCallback, useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
-import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
+import { logPageViewEvent, logViewEvent } from '../../../lib/metrics';
 import { useAccount } from '../../../models';
 import { FtlMsg } from 'fxa-react/lib/utils';
 import { useFtlMsgResolver } from '../../../models/hooks';
 
 import { InputText } from '../../../components/InputText';
 import CardHeader from '../../../components/CardHeader';
 import WarningMessage from '../../../components/WarningMessage';
-import { LinkExpiredResetPassword } from '../../../components/LinkExpiredResetPassword';
-import { ResetPasswordLinkDamaged } from '../../../components/LinkDamaged';
 import { LinkStatus, MozServices } from '../../../lib/types';
 import { REACT_ENTRYPOINT } from '../../../constants';
-import {
-  RequiredParamsAccountRecoveryConfirmKey,
-  useAccountRecoveryConfirmKeyLinkStatus,
```

Comment:
Forgot to call this out in previous review, can we nix this `useLinkStatus` file now that we don't need it any longer? You can add `RequiredParamsAccountRecoveryConfirmKey` as a basic object similar to `type CompleteResetPasswordParams` in CompleteResetPassword.

Reply:
Sweet, I'll remove that. I don't see any of that hook's exports in use except here.